src/python/WMCore/Services/RucioConMon/RucioConMon.py

```diff
@@ -103,11 +122,6 @@ def getRSEUnmerged(self, rseName, zipped=False):
             rseUnmerged = self._getResult(uri, callname=rseName)
             return rseUnmerged
         else:
-            pass
-            # TODO: To implement the _getResultZipped() method
-            # NOTE: An alternative uri - providing the file with .zip extension:
-            #       uri = "WM/files/files.gz?rse=%s&format=raw" % rseName
-            #       The uri from below provides the file zipped but with no extension
-            # uri = "WM/files?rse=%s&format=raw" % rseName
-            # rseUnmerged = self._getResultZipped(rseName,  callname='unmerged.zipped', clearCache=True)
-            # return rseUnmerged
+            uri = "WM/files?rse=%s&format=raw" % rseName
+            rseUnmerged = self._getResultZipped(uri,  callname='unmerged.zipped', clearCache=True)
```

Comment:
I think we'd better  include the `rseName` in the caller's alias creation, so that we are sure we are using separate caches for different RSEs. Similar to how it is done above.

Reply:
done